chainer/functions/connection/n_step_lstm.py

```diff
@@ -453,7 +453,7 @@ def n_step_lstm(
 
     """
 
-    xp = cuda.get_array_module(hx.data)
+    xp = cuda.get_array_module(hx, hx.data)
```

Comment:
I did not understand that your intension is that it works `hx` is both `ndarray` or `Variable`. Now I come to think this change is acceptable.